code/modules/surgery/amputation.dm

```diff
@@ -14,7 +14,7 @@
 
 
 /datum/surgery_step/sever_limb
-	name = "sever limb"
+	name = "sever limb (shears/scalpel/saw)"
```

Comment:
```suggestion
	name = "sever limb (circular saw)"
```

Show the tool we actually want people to use (it's weird to me scalpel is 100% accuracy)